library/packages/src/modules/SignatureCheckDialogs.rb

```diff
@@ -878,10 +876,10 @@ def HandleDoNotShowDialogAgain(default_return, dont_show_dialog_ident, dont_show
 
     # Function adds delimiter between after_chars characters in the string
     #
-    # @param string to be splitted
-    # @param [String] delimiter
-    # @param integer after characters
-    # @return [String] with delimiters
+    # @param whattosplit [String] text to be splitted
```

Comment:
I try to be consistent with rest of file, somewhere ( at start I try to modify it, but it is huge amount of places). I also prefer first form 😉